src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -224,6 +224,11 @@ object Parameters {
     "internal_support_aws_stage_end_point"
   )
 
+  // Support to share JDBC connection if possible
```

Comment:
We introduce 2 methods to disable shared connection:
1. Spark Connector option: `support_share_connection` ( default is true)
2. Static function: SparkConnectorContext.disableSharedConnection()

In most cases, customer may create a sfOptions map and use it in everywhere.
I suppose method 1 is used in most cases.

Method 2 can be used if user can't update sfOptions easily.